view_all_set.php

```diff
@@ -122,6 +122,7 @@
 			exit; # stop here
 		}
 	}
+	$t_setting_arr = filter_ensure_valid_filter( $t_setting_arr );
```

Comment:
yes, an else branch there makes more sense.
as it is now, it would create an empty filter, that would not be used anyway.